File: sign/sign-interface/src/main/java/com/drzyl/sign/vo/RoleVO.java
package com.drzyl.sign.vo;

import java.io.Serializable;

import lombok.Data;

/**
 * 角色 SYS_ROLE 展示类
 */
@Data
public class RoleVO implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String roleId; // 角色编号

  protected String roleName; // 角色名称

  protected String roleType; // 角色类型

  protected String accessList; // 权限列表

  protected Integer weightOrder; // 排序权重

  protected Integer versions; // 版本号

  protected String createId; // 创建人账号

  protected String createName; // 创建人名称

  protected Long createTimes; // 创建时间

  protected String modifyId; // 修改人账号

  protected String modifyName; // 修改人名称

  protected Long modifyTimes; // 修改时间

}

File: vue-coalsale/src/view/com/drzyl/sign/role/RoleVO.js
export default class RoleVO {
  constructor () {
    this.roleId = null // 角色编号
    this.roleName = null // 角色名称
    this.roleType = null // 角色类型
    this.accessList = null // 权限列表
    this.weightOrder = null // 排序权重
    this.versions = null // 版本号
    this.createId = null // 创建人账号
    this.createName = null // 创建人名称
    this.createTimes = null // 创建时间
    this.modifyId = null // 修改人账号
    this.modifyName = null // 修改人名称
    this.modifyTimes = null // 修改时间
  }
}
